.github/actions/run-integration-tests/action.yml

```diff
@@ -2,14 +2,18 @@ name: "Run integration tests"
 description: "Run Inspektor Gadget integration tests"
 
 inputs:
+  push:
+    description: 'Defines whether the generated integration tests image has to be push to the container registry or not'
+    required: true
+    default: 'false'
```

Comment:
What about using boolean [`false`](https://docs.github.com/en/actions/learn-github-actions/expressions#literals) instead?
Then, you do not need later to compare string.

Reply:
It doesn't work actually, see https://github.com/actions/runner/issues/1483. There are some workarounds but I found it easier to just check against `'true'`.